src/googleapis.gmail/gmail_data_mappings.bal

```diff
@@ -240,7 +240,7 @@ function convertJSONToHeaderMap(json jsonMsgPartHeaders) returns map<string> {
 # + return - Label type object
 function convertJSONToLabelType(json sourceLabelJsonObject) returns Label|error {
     Label targetLabel = {};
-    map<json>|error srcLabelJsonObjectMap = map<json>.constructFrom(sourceLabelJsonObject);
+    map<json>|error srcLabelJsonObjectMap = <map<json>|error> sourceLabelJsonObject.cloneWithType(json);
```

Comment:
Cant we directly use map<json> as the type instead of JSON?

Reply:
Added the change to all the locations.